From 087e5634f97076d629a8a03d6eda779bffced3cb Mon Sep 17 00:00:00 2001
From: Mark Brown <broonie@linaro.org>
Date: Fri, 13 Jun 2014 21:36:34 +0100
Subject: [PATCH] mailbox: Prototype mbox_client_peek_data()

This is an interface intended for client drivers and exported but it is
not prototyped in the headers so can't be used. Add a prototype.

Signed-off-by: Mark Brown <broonie@linaro.org>
---
 include/linux/mailbox_client.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/mailbox_client.h b/include/linux/mailbox_client.h
index 53eb078..cbfcf84 100644
--- a/include/linux/mailbox_client.h
+++ b/include/linux/mailbox_client.h
@@ -40,6 +40,7 @@ struct mbox_client {
 struct mbox_chan *mbox_request_channel(const struct mbox_client *cl);
 int mbox_send_message(struct mbox_chan *chan, void *mssg);
 void mbox_client_txdone(struct mbox_chan *chan, int r);
+bool mbox_client_peek_data(struct mbox_chan *chan);
 void mbox_free_channel(struct mbox_chan *chan);
 
 #endif /* __MAILBOX_CLIENT_H */
-- 
1.8.0

